### ISSUE TITLE
KindleGen failed to create MOBI!

### ISSUE THREAD

**Issue by Boco-T** [ciromattia/kcc#702]
Error: KindleGen failed to create MOBI!
[1, 'Error(opfparser):E34002: Please specify the Original-resolution metadata value in the format WIDTH x HEIGHT.', 'file.epub']

Trying to convert 3 comics to MOBI to load on a Kindle Fire. Running command line version and getting these results:

D:\Books\Kindle Convertor>kcc_c2e_6.0.0.exe -r 1 -c 0 --forcecolor --mozjpeg -f MOBI --customwidth 800 --customheight 1280 "book v07"
comic2ebook v6.0.0 - Written by Ciro Mattia Gonano and Pawel Jastrzebski.
Working on book v07...
Preparing source images...
Checking images...
Processing images...
Creating EPUB file...
Creating MOBI files...
Error: KindleGen failed to create MOBI!
[1, 'Error(opfparser):E34002: Please specify the Original-resolution metadata value in the format WIDTH x HEIGHT.', 'book v07.epub']

The first book worked fine and made the MOBI and it looks great, but the other two both return this same error. EPUB seems fine but I need it in MOBI for the Kindle Fire to open it natively.

**Comment by axu2** [comment]
For a Fire Tablet, you should use a CBZ reader app like Koreader.

**Comment by Boco-T** [comment]
I have no problem with that, I use ComicScreen. I'm trying to make a Kindle file for my partner so I don't have to put a separate program onto her Kindle. It's not really relevant to this report, all I want to know is what that error means and why one folder of images worked fine but two others didn't.

**Comment by axu2** [comment]
Can you replicate the issue with the GUI version of KCC?

**Comment by Boco-T** [comment]
Is there a way to do the specific resolution in the GUI version? I went to the command line because I wanted to specify 800x1280 (the resolution of the Kindle Fire I am putting the book on) and none of the pre-built profiles have that resolution.

I did run it selecting "Kobo Aura HD" in the GUI and it worked, see attached.
![image](https://github.com/ciromattia/kcc/assets/99706363/893ad9f9-06ee-41a7-825d-7716279b72b4)
